include/solvers/navier_stokes_scratch_data.h

```diff
@@ -686,10 +686,10 @@ class NavierStokesScratchData
     fe_values_local_particles.reinit(cell);
 
     fe_values_local_particles[velocities].get_function_laplacians(
-      current_solution, fluid_velocity_laplacian_at_particle_location);
+      previous_solution, fluid_velocity_laplacian_at_particle_location);
 
     fe_values_local_particles[pressure].get_function_gradients(
-      current_solution, fluid_pressure_gradients_at_particle_location);
+      previous_solution, fluid_pressure_gradients_at_particle_location);
```

Comment:
Why use previous solutions instead of current one here?
Is it because it does not converge if you use current one?

Reply:
Yes, when using model A, this the shear force is not applied and thus there is no problem. However, for model B, it stagnates at a residue of 10^-5 if I use the current solution. Using the previous solution allows it to converge quickly.

Reply:
Ok I understand, we have to understand that this will generate a small difference in the results between A and B :). Good. I'll proceed and merge then!